pkg/client/podspecreview.go

```diff
@@ -0,0 +1,33 @@
+package client
+
+import securityapi "github.com/openshift/origin/pkg/security/api"
+
+// PodSpecReviews has methods to work with PodSpecReview resources in the cluster scope
+type PodSpecReviews interface {
+	PodSpecReviews() PodSpecReviewInterface
```

Comment:
These are namespaced resources, so the top interface should be `PodSpecReviewNamespacer` which takes a namespace